bin/deploy-to-prod.sh

```diff
@@ -4,9 +4,7 @@ if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ $(phpenv version-name) = "7.0" ];
     eval "$(ssh-agent -s)"
     chmod 600 .travis/deploy.key
     ssh-add .travis/deploy.key
-    mkdir build
-    mv * build
-    tar -czf package.tgz build
     scp package.tgz $DEPLOY_USER@$DEPLOY_HOST:$DEPLOY_PATH
     ssh $DEPLOY_USER@$DEPLOY_HOST $DEPLOY_PATH/deploy.sh
+    ssh php ~/php.of.by/bin/console doctrine:migrations:migrate
```

Comment:
Looks like missed credential
`ssh php ~/php.of.by/bin/console doctrine:migrations:migrate`

Should be 
`ssh $DEPLOY_USER@$DEPLOY_HOST php ~/php.of.by/bin/console doctrine:migrations:migrate`